ui/src/app/views/workspace-edit/component.html

```diff
@@ -85,8 +86,9 @@
       </div>
     </section>
   </div>
-  <div class="button">
+  <div style="display: inline-block" class="button">
```

Comment:
We will want to be careful with `display: inline-block` generally. It often creates very odd vertical whitespace that appears to have no explanation.

Reply:
I can move this to a flex container.